### PR TITLE
Feature/#156 partner profile update

### PR DESCRIPTION
**Pull request by kenzo23312**
PATCH sections

### CODE REVIEW

File: frontend/src/reducers/conceptNote.js [hunk 1]
```diff
@@ -52,6 +52,7 @@ export const projectApplicationExists = partnerId => (dispatch) => {
       dispatch(uploadCnSuccess(profiles));
     })
     .catch((error) => {
+      debugger
```
Comment (marcindo) [inline frontend/src/reducers/conceptNote.js]: debugger

File: frontend/src/reducers/conceptNote.js [hunk 2]
```diff
@@ -84,6 +85,7 @@ export const uploadPartnerConceptNote = projectId => (dispatch, getState) => {
       dispatch(selectLocalCnFile(null));
     })
     .catch((error) => {
+      debugger
```
Comment (marcindo) [inline frontend/src/reducers/conceptNote.js]: debugger

File: frontend/src/components/organizationProfile/edit/funding/partnerProfileFunding.js [hunk 3]
```diff
@@ -17,18 +26,91 @@ const STEPS = readOnly => [
...
+    this.setState({ actionOnSubmit: 'exit' });
+  }
+
+  handleSubmit(formValues) {
+    const { initialValues, updateTab, partnerId, loadPartnerProfileDetails } = this.props;
+
+    const funding = flatten(formValues.fund);
+    const initFunding = flatten(initialValues.fund);
+
+    return updateTab(partnerId, 'funding', changedValues(initFunding, funding))
```
Comment (marcindo) [inline frontend/src/components/organizationProfile/edit/funding/partnerProfileFunding.js]: I think the then and catch are always the same whenever this function appears, so would be better to extract them as some utils

File: frontend/src/reducers/conceptNote.js (hunk 1 above)
Comment (kenzo23312) [reply frontend/src/reducers/conceptNote.js]: done

File: frontend/src/reducers/conceptNote.js (hunk 2 above)
Comment (kenzo23312) [reply frontend/src/reducers/conceptNote.js]: done

**Pull request merged**